man/zeek.8

```diff
@@ -16,6 +16,8 @@ tasks, including detecting malware by interfacing to external registries,
 reporting vulnerable versions of software seen on the network, identifying
 popular web applications, detecting SSH brute-forcing, validating SSL
 certificate chains, among others.
+
+You must have read access to the files or interfaces specified.
```

Comment:
Hm. I am stumbling over this a little bit since it is not exactly accurate - since you can, e.g., also use setcap on the zeek binary - in whch case you do no longer need access to the interface.

Since having adequate permissions to the interface (or files) you read is generally a requirement which, I don't think is typically called out in manpages - would you feel very bad if we just dropped this?

Alternatively - I would also be open to do it similarly to how the tcpdump documentation mentions this. But that is way more detailed.

Reply:
Hmm, yeah, it can also vary among systems. 

OpenBSD tcpdump says, "You must have read access to /dev/bpf."

Linux tcpdump says:
"       Reading packets from a network interface may require that you have special privileges; see
       the  pcap  (3PCAP) man page for details.  Reading a saved packet file doesn't require spe‐
       cial privileges."
I dont feel strongly about this line. Make changes as you wish.